packages/manager/src/components/Autocomplete/Autocomplete.styles.tsx

```diff
@@ -58,9 +58,13 @@ export const CustomPopper = (props: PopperProps) => {
   return (
     <Popper
       {...rest}
+      modifiers={[
+        { enabled: false, name: 'preventOverflow' },
+        { enabled: !placement, name: 'flip' },
+      ]}
       data-qa-autocomplete-popper
       data-testid="autocomplete-popper"
-      modifiers={[{ enabled: false, name: 'preventOverflow' }]}
+      placement={placement || undefined}
```

Comment:
why this or statement. Won't be this placement value will be undefined by default if nothing passed?